plugins-compat-tester/src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -426,6 +426,7 @@ private TestExecutionResult testPluginAgainst(MavenCoordinates coreCoordinates,
             forExecutionHooks.put("pomData", pomData);
             forExecutionHooks.put("pom", pom);
             forExecutionHooks.put("coreCoordinates", coreCoordinates);
+            forExecutionHooks.put("pluginCheckoutDir", pluginCheckoutDir);
```

Comment:
This is needed for hooks that look at the `pluginCheckoutDir` which is needed for a fix/hook that coincides with this change.